include/pipe.h

```diff
@@ -0,0 +1,41 @@
+#include <mutex.h>
```

Comment:
You're missing header file prologue and epilogue here. Please look at other files to align your style.